.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ orbs:
 jobs:
   build:
     machine:
-      image: ubuntu-1604:201903-01
+      image: ubuntu-2004:202201-02
```

Comment:
Can we update here as well?

https://github.com/mathworks/ci-configuration-examples#:~:text=image%3A%20ubuntu%2D2004%3A202107%2D02